client/app/assets/less/inc/schema-browser.less

```diff
@@ -72,8 +72,14 @@ div.table-name {
 
 .schema-control {
   display: flex;
+  flex-wrap: nowrap;
   padding: 0;
 
+  button {
```

Comment:
Is there any other `<button>` or this refers to the Ant Button?

Reply:
Ant Button. Replaced this with `.ant-btn`